### ISSUE TITLE
Dock gets stuck, sometimes unpredictably

### ISSUE THREAD

**Issue by MakeMike** [micheleg/dash-to-dock#2209]
Hey folks! I've encountered an issue on Dash to Dock where the dock gets stuck and the screen around the dock doesn't respond to my mouse clicks. It happens randomly, but I can force it to happen when I rearrange the apps.

dconf dump /org/gnome/desktop/interface/
[/]
clock-format='12h'
cursor-size=24
cursor-theme='Sunity-cursors'
document-font-name='SF Pro Display 11'
enable-animations=true
enable-hot-corners=false
font-antialiasing='grayscale'
font-hinting='slight'
font-name='Noto Sans,  10'
gtk-theme='White'
icon-theme='Nordzy--dark_panel'
monospace-font-name='FiraCode Nerd Font Mono weight=450 10'
scaling-factor=uint32 1
text-scaling-factor=1.0
toolbar-style='text'
toolkit-accessibility=false

dconf dump /org/gnome/shell/extensions/dash-to-dock/
[/]
always-center-icons=false
animation-time=0.10000000000000002
apply-custom-theme=false
autohide=true
background-color='rgb(13,15,18)'
background-opacity=0.40000000000000002
custom-background-color=true
dash-max-icon-size=48
dock-position='BOTTOM'
extend-height=false
height-fraction=0.90000000000000002
hide-delay=0.10000000000000002
hot-keys=false
intellihide-mode='ALL_WINDOWS'
preferred-monitor=-2
preferred-monitor-by-connector='HDMI-A-0'
pressure-threshold=50.0
preview-size-scale=0.0
require-pressure-to-show=true
transparency-mode='FIXED'

**Comment by vanvugt** [comment]
Does the top panel still respond to clicks?

**Comment by MakeMike** [comment]
Yes it does, the only thing that seems unresponsive is the area on the left
and right of the dock.

**Comment by Victorhtf** [comment]
Same for me. If I open menu and close it again the dock responds disappearing, as it should do.
